doc/source/security/data-scrubbing.rst

```diff
@@ -0,0 +1,54 @@
+.. _data-at-rest-encryption.data-scrubbing:
+.. _data-scrubbing:
+
+Data Scrubbing
```

Comment:
About Data Scrubbing - you need to reach out to George and Zsolt to check whether they are going a version of it.

Reply:
Will do.